.circleci/config.yml

```diff
@@ -670,10 +688,18 @@ workflows:
         use_local_kuma_images: true
     - e2e:
         <<: *commit_workflow_filters
-        name: test/e2e
+        name: test/e2e-ipv4
+        requires:
+          - images
+          - check
+    - e2e:
+        <<: *commit_workflow_filters
+        name: test/e2e-ipv6
         requires:
           - images
           - check
+        # custom parameters
+        ipv6: true
```

Comment:
so we will be running twice as much E2E on commit? Should this be maybe run on master?

Reply:
Yes, but this will come after we merge https://github.com/kumahq/kuma/pull/1684